docs/components/FigmaLogo.tsx

```diff
@@ -0,0 +1,32 @@
+export const FigmaLogo = () => {
+	return (
+		<svg
```

Comment:
```suggestion
		<svg
			width="24"
			height="24"
			viewBox="0 0 24 24"
			fill="none"
			xmlns="http://www.w3.org/2000/svg"
			aria-hidden="true"
			focusable="false"
		>
			<path
				d="M11.5 12C11.5 9.81338 13.3834 8.04071 15.7067 8.04071C18.03 8.04071 19.9135 9.81338 19.9135 12C19.9135 14.1867 18.03 15.9593 15.7067 15.9593C13.3834 15.9593 11.5 14.1867 11.5 12Z"
				fill="#1ABCFE"
			/>
			<path
				d="M3.08643 19.9186C3.08643 17.732 4.96984 15.9593 7.29316 15.9593H11.4999V19.9186C11.4999 22.1052 9.61649 23.8778 7.29316 23.8778C4.96984 23.8778 3.08643 22.1052 3.08643 19.9186Z"
				fill="#0ACF83"
			/>
			<path
				d="M11.5 0.122192V8.04072H15.7067C18.0301 8.04072 19.9135 6.26811 19.9135 4.08147C19.9135 1.89482 18.0301 0.122192 15.7067 0.122192H11.5Z"
				fill="#FF7262"
			/>
			<path
				d="M3.08643 4.08147C3.08643 6.26811 4.96984 8.04072 7.29316 8.04072H11.4999V0.122192H7.29316C4.96984 0.122192 3.08643 1.89482 3.08643 4.08147Z"
				fill="#F24E1E"
			/>
			<path
				d="M3.08643 12C3.08643 14.1867 4.96984 15.9593 7.29316 15.9593H11.4999V8.04071H7.29316C4.96984 8.04071 3.08643 9.81338 3.08643 12Z"
				fill="#A259FF"
			/>
		</svg>
```

Reply:
Our button icons should be 24x24, right now this icon is only 16x16. Also the viewbox isn't a square so I think it might look odd in some browers?